package.json

```diff
@@ -1,14 +1,12 @@
 {
 	"name": "@silverstripe/webpack-config",
-	"version": "0.4.1",
+	"version": "0.5.0",
 	"description": "SilverStripe config files for modules",
 	"engines": {
 		"node": ">= 6.x"
 	},
 	"bin": {
-		"webpack": "./node_modules/.bin/webpack",
-		"sass-lint": "./node_modules/.bin/sass-lint",
```

Comment:
Isn't the task to move only eslint? Why removing sasslint as well? And this will give an error when running `yarn lint`.

Reply:
Good catch. I think I had started down the path of breaking out `sass-lint`, too, and then changed my mind. ;-)

Reply:
hahaha sweet! Will test it again.